ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -5,7 +5,7 @@
 
 <SyncHeader
   @title="Sync Secrets to {{@destination.name}}"
-  @icon={{@destination.icon}}
+  @icon={{get (find-by "type" @destination.type (sync-destinations)) "icon"}}
```

Comment:
seems like this could be added to the `sync-destinations` helper, but it's simple enough template logic this works as is

Reply:
Ya I looked at that and it's odd. There is a `findDestination` util in that file which is used in a bunch of places but the actual template helper that is exported just returns the full destinations map. Not sure what 2 years ago me was thinking 🤷 . I thought about making another helper but landed on this since it's the only place I've run into it so far.

Reply:
It was here too https://github.com/hashicorp/vault/pull/30554/files#diff-e35ae22bf7e30afdf79f1f679b477fa6a5ed9910c728ca8627c70046ae061f5cR7 which is what made me think to call it out

Reply:
Oh ya I guess it is. Too many changes I can't keep track! Maybe I'll revisit this during cleanup.